src/machine/behavior/is/memory/ldbu.rs

```diff
@@ -1,6 +1,14 @@
 use machine::state::State;
+use machine::state::mem::ByteAt;
 
-pub fn ldbu(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn ldbu(state: &mut State, x: u8, y: u8, z: u8) {
+
+    let op1: u64 = state.gpr[y].into();
+    let op2: u64 = state.gpr[z].into();
+    let mut a = op1 + op2;
+    a = a % 2u64.pow(64);
```

Comment:
What about [`u64::wrapping_add()`](https://doc.rust-lang.org/std/primitive.u64.html#method.wrapping_add) or [`u64::overflowing_add()`](https://doc.rust-lang.org/std/primitive.u64.html#method.overflowing_add)? You don't necessarily need to pull in another crate for that. The `Wrapping` type might be a good idea to reduce code fluff though... I can't tell, you have to decide ;-)